checks/cs50/2017/x/speller/checks.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import *
+
+class Speller(Checks):
+
+    @check()
+    def exists(self):
+        """dictionary.c, dictionary.h, and Makefile exist"""
+        super(Speller, self).exists("dictionary.c")
+        super(Speller, self).exists("dictionary.h")
+        super(Speller, self).exists("Makefile")
+
+    @check("exists")
+    def compiles(self):
+        """speller compiles"""
+        self.include("speller.c")
```

Comment:
Hm, though in Python, `import` is pretty similar in spirit, so far as TFs would perceive it, as `include` in C. I think we used to use `copy`? We could then support an optional second argument (destination file name) so that, if we have multiple `speller.c` files (in some other scenario), we can name them in some self-descriptive way but copy them into the student's dir as just `speller.c`?